SpywareFilter/sections/specific.txt

```diff
@@ -60,6 +60,7 @@ news.mynavi.jp##.articleList-attention
 ||content.viralize.tv/track/
 /track-internal-links.js$domain=balkangunlugu.com.tr
 ||tv-tokyo.co.jp/index/scatalyst/
+||bildirt.com/BildirtSDKfiles.js
```

Comment:
This URL is already blocked by `||bildirt.com^$third-party` of Annoyance filter.